src/pages/Admin/Overview/index.jsx

```diff
@@ -250,8 +249,9 @@ export default class Overview extends React.Component {
   };
 
   developerView = () => {
-    const { appStore, clusterStore, repoStore, runtimeStore, t } = this.props;
+    const { appStore, clusterStore, repoStore, runtimeStore, t, i18n } = this.props;
     const { isLoading } = appStore;
+    const lng = i18n.language || 'zh';
```

Comment:
No need to pass `lng` params, `getPastTime ` can read it from localStorage

Reply:
ok